test/e2e/framework/util.go

```diff
@@ -177,6 +177,10 @@ const (
 	// How long PVs have to become deleted
 	PVDeletingTimeout = 3 * time.Minute
 
+	// How long a node is allowed to become "Ready" after it is recreated before
+	// the test is considered failed.
+	RecreateNodeReadyAgainTimeout = 6 * time.Minute
```

Comment:
Sometimes node recreation may take longer time. Maybe make it 10 minutes to be safe?